lib/ui/playground/data/playground_tasks_datasource.dart

```diff
@@ -8,8 +11,21 @@ class PlayGroundTasksDataSource implements PlaygroundDataSource {
     return [
       //Add Your tasks here
       PlaygroundWidget(
-        title: 'TASK FARM-402 Test playground widget',
-        child: Text('Test play ground'),
+        title: 'FARM-355 Generic Action Sheet - Type 1',
+        child: ActionSheet(viewModel: MockActionSheetViewModel.buildStandard(), style: ActionSheetStyle.defaultStyle())
```

Comment:
Add a Container as a parent of ActionSheet with height to display the ActionSheet like the real one. Do the same with the other ActionSheet in this class